src/transformers/modeling_longformer.py

```diff
@@ -1504,7 +1526,7 @@ def forward(
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
         # set global attention on question tokens
-        if global_attention_mask is None:
+        if global_attention_mask is None and input_ids is not None:
```

Comment:
What is use case for `input_ids=None`?

Reply:
just don't do it -> the user has to set the mask himself then.